integration_test/ops_agent_test/main_test.go

```diff
@@ -4435,7 +4435,7 @@ func getRecentServiceOutputForImage(imageSpec string) string {
 		}, ";")
 		return cmd
 	}
-	return "sudo systemctl status google-cloud-ops-agent"
+	return "sudo cat /var/log/google-cloud-ops-agent/health-checks.log && sudo systemctl status google-cloud-ops-agent"
```

Comment:
Why are we moving health checks to their own log file? `systemctl status` is where users expect to see error messages.